app/services/onlinetesting/OnlineTestService.scala

```diff
@@ -328,9 +325,11 @@ trait OnlineTestService {
   }
 
   def markAsStarted(cubiksUserId: Int): Future[Unit] = {
```

Comment:
I think "markAsStarted" that should include phase1 in the name, maybe "markAsPhase1Started"